server/src/main/scala/org/http4s/server/staticcontent/WebjarService.scala

```diff
@@ -55,7 +55,9 @@ object WebjarService {
     * @param config The configuration for this service
     * @return The HttpRoutes
     */
-  def apply[F[_]](config: Config[F])(implicit F: Sync[F], cs: ContextShift[F]): HttpRoutes[F] = {
+  def apply[F[_]](config: Config[F], classLoader: Option[ClassLoader] = None)(
```

Comment:
This is not binary compatible, so we'll have to relegate this to master. (And we need to figure out why our MiMa is not running.  This burned us on 0.20.22.)

Reply:
oh okay, do you want me to raise this PR against master?

Reply:
Yes, unless we can figure out a way to make it binary compatible.  We could if we left the original signature, and called the new one something other than `apply`.  Like `withClassLoader`.  But we'd probably want to deprecate that in 1.0 in favor of the builder.